client/milvusclient/read_options.go

```diff
@@ -102,10 +102,13 @@ func (r *AnnRequest) searchRequest() (*milvuspb.SearchRequest, error) {
 
 	var err error
 	// placeholder group
-	request.PlaceholderGroup, err = vector2PlaceholderGroupBytes(r.vectors)
+	placeHolderGroupBytes, err := vector2PlaceholderGroupBytes(r.vectors)
 	if err != nil {
 		return nil, err
 	}
+	request.SearchInput = &milvuspb.SearchRequest_PlaceholderGroup{
```

Comment:
whe don't support binary vector for restful?

Reply:
read_options is for milvus client instead of restfult.